bin.src/makeLSSTobs.py

```diff
@@ -13,21 +13,49 @@
 from lsst.sims.maf.batches import getColMap
 
 
-def readOpsim(opsimfile, constraint=None, dbcols=None):
+def readOpsim(opsimfile, constraint=None, footprint='camera', dbcols=None):
```

Comment:
Hmm. I suppose. It's really only there so that I could decide if 'rotSkyPos' was a required column or not (I'd like to just say it's required, but Bryce's old subset of a opsim database didn't have it, so I couldn't just do that).

Reply:
Added documentation to makeLSSTobs.py methods.